### DISCUSSION TITLE
Feature progress

### DISCUSSION THREAD

**Discussion by srothetafesa** [srothetafesa/Tafe24S1T109_Ext#7] [General]
Hi John,

Just letting you know I have finished my feature for the universal calculator. Let me know when you are ready and we can proceed with merging our feature branches into the development branch.

Thanks,
Sam

**Comment by JohnLSison** [comment]
Hey Sam,

Almost done on my end just doing one more test to ensure that the main menu button directs me to my converter calculator and I will do one final test to check if all conversions are working.

Cheers,
John
